docs/Salesforce-batchsink.md

```diff
@@ -11,14 +11,19 @@ Currently, only inserts are supported. Upserts are not supported.
 Configuration
 -------------
 
+**Use Connection:** Whether to use a connection. If a connection is used, you do not need to provide the credentials.
+
+**Connection:** Name of the connection to use. Object Names information will be provided by the connection.
+You also can use the macro function ${conn(connection-name)}.  
+
 **Reference Name:** Name used to uniquely identify this sink for lineage, annotating metadata, etc.
 
 **Username:** Salesforce username.
 
 **Password:** Salesforce password.
 
-**Security Token:** Salesforce security Token. If the password does not contain the security token the plugin 
-will append the token before authenticating with salesforce.
+**Security Token:** Salesforce security Token. If the password does not contain the security token, the plugin 
```

Comment:
Should it say Salesforce Security Token?

Reply:
changed to Salesforce security token